src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super T> machines, T value);
```

Comment:
The `fill` method is correctly using `List<? super T>` for the `machines` parameter, which allows adding elements of type `T` or its superclasses. This aligns with the requirement to replace `Object` with a parameterized value.